sys-process/vixie-cron/vixie-cron-4.1-r15.ebuild

```diff
@@ -16,7 +16,7 @@ SRC_URI="mirror://gentoo/${P}.tar.bz2
 	mirror://gentoo/${P}-gentoo-${GENTOO_PATCH_REV}.patch.bz2"
 
 LICENSE="ISC BSD-2 BSD"
-KEYWORDS="alpha amd64 arm arm64 hppa ia64 m68k ~mips ppc ppc64 s390 sh sparc x86 ~amd64-fbsd ~x86-fbsd"
+KEYWORDS="alpha amd64 arm arm64 hppa ia64 m68k ~mips ppc ppc64 ~riscv s390 sh sparc x86 ~amd64-fbsd ~x86-fbsd"
```

Comment:
`sys-process/vixie-cron` is no longer maintained by upstream, and currently `sys-process/cronie` is recommended, so you might consider to test `sys-process/cronie` and add `~riscv` there.
In either case, you might add `~riscv` to `virtual/cron`.